src/encoded/types/reference_genome.py

```diff
@@ -16,4 +19,42 @@
 class ReferenceGenome(Item):
     item_type = "reference_genome"
     schema = load_schema("encoded:schemas/reference_genome.json")
-    embedded_list = []
+    embedded_list = _build_reference_genome_embedded_list()
+
+    def get(self, name, default=None):
+        """Override method to allow submitted_id keys for reference genome.
+
+        Allows DonorSpecificAssembly types to show up in search of RefrenceGenome collection.
+        """
+        resource = super(Item, self).get(name, None)
+        if resource is not None:
+            return resource
+        if is_accession(name):
+            resource = self.connection.get_by_unique_key("accession", name)
+            if resource is not None:
+                if not self._allow_contained(resource):
+                    return default
+                return resource
+        if ":" in name:
+            resource = self.connection.get_by_unique_key("alias", name)
+            if resource is not None:
+                if not self._allow_contained(resource):
+                    return default
+                return resource
+        if "DONOR-SPECIFIC-ASSEMBLY" in name:
+            resource = self.connection.get_by_unique_key("submitted_id", name)
+            if resource is not None:
+                if not self._allow_contained(resource):
+                    return default
+                return resource
```

Comment:
I think everything in this function, except this section, is done as part of L29 `super(Item, self).get(name, None)`, so I think you can simplify this to just these lines?